infrastructure-playbooks/shrink-osd.yml

```diff
@@ -218,6 +218,22 @@
       loop: "{{ _osd_hosts }}"
       when: item.2 in _lvm_list.keys()
 
+    - name: refresh lvm metadata cache
+      when:
+        - containerized_deployment | bool
+        - ansible_os_family == 'RedHat'
```

Comment:
not sure this really makes sense in master branch (maybe it should be added as part of the backport)